docs/source/membership/membership.md

```diff
@@ -50,62 +94,200 @@ multiple MSPs and name them accordingly, e.g., `ORG2-MSP-NATIONAL` and
 `ORG2` in the `NATIONAL` sales channel compared to the `GOVERNMENT` regulatory
 channel.
 
-![MSP1](./membership.diagram.3.png)
+![MSP1b](./membership.diagram.3.png)
 
 *Two different MSP configurations for an organization. The first configuration shows
 the typical relationship between an MSP and an organization --- a single MSP defines
 the list of members of an organization. In the second configuration, different MSPs
 are used to represent different organizational groups with national, international,
 and governmental affiliation.*
 
-#### Organizational Units and MSPs
-
-An organization is often divided up into multiple **organizational units** (OUs), each
-of which has a certain set of responsibilities. For example, the `ORG1`
-organization might have both `ORG1-MANUFACTURING` and `ORG1-DISTRIBUTION` OUs
-to reflect these separate lines of business. When a CA issues X.509 certificates,
-the `OU` field in the certificate specifies the line of business to which the
-identity belongs.
-
-We'll see later how OUs can be helpful to control the parts of an organization who
-are considered to be the members of a blockchain network. For example, only
-identities from the `ORG1-MANUFACTURING` OU might be able to access a channel,
-whereas `ORG1-DISTRIBUTION` cannot.
+### Organizational Units (OUs) and MSPs
+
+An organization can also be divided into multiple **organizational units**, each
+of which has a certain set of responsibilities. Think of an OU as a department
+inside an organization. For example, the `ORG1` organization might have both
+`ORG1.MANUFACTURING` and `ORG1.DISTRIBUTION` OUs to reflect these separate
+lines of business. When a CA issues X.509 certificates, the `OU` field in
+the certificate specifies the line of business to which the identity
+belongs. These values can then be used in smart contracts for
+attribute based access control.
+
+Specifying OUs is optional. If no OUs are listed, all the identities that are part of
+an MSP --- as identified by the Root CA and Intermediate CA folders --- will be considered
+members of the organization.
+
+### Node OU Roles and MSPs
+
+Additionally, there is a special kind of OU, sometimes referred to as a
+Node OU, that can be used to confer a role onto an identity. These
+Node OU roles are defined in the `$FABRIC_CFG_PATH/msp/config.yaml`
+file and contain a list of organizational units whose members are considered
+to be part of the organization represented by this MSP. This is particularly
+useful when you want to restrict the members of an organization to the ones
+holding an identity (signed by one of MSP designated CAs) with a specific
+Node OU role in it. For example, an endorsement policy can specify that
+only members of ORG1 can endorse a transaction. But when Node OU roles are
+enabled, a more granular restriction can be implemented,
+requiring that only peers in ORG1 can endorse a transaction instead of
+any member of ORG1.
+
```

Comment:
I'm a little reluctuant to introduce another term here, but I get what you are saying. Role is used by the Fabric CA so I was trying to stick with that since it does map -- especially along the lines of the OU discussion. At least that is where I am coming from

Reply:
For your previous comment I added this:
How is that chain of trust used by the blockchain network?

Certificate Authorities issue identities by generating a public and private key which forms a key-pair that can be used to prove identity. Because a private key can never be shared publicly, a mechanism is required to enable that proof which is where the MSP comes in. For example, a peer uses its private key to digitally sign, or endorse, a transaction.  The MSP on the ordering service contains the peer's public key which is then used to verify that the signature attached to the transaction is valid. The private key is used to produce a signature on a transaction that only the corresponding public key, that is part of an MSP, can match. Thus, the MSP is the mechanism that allows that identity to be trusted and recognized by the rest of the network without ever revealing the member’s private key.